bin/pry

```diff
@@ -17,6 +19,10 @@ def udec(*)
   UBID.decode(*)
 end
 
+def udec_deleted(ubid)
+  DeletedRecord.where(Sequel.pg_json(:model_values)["name"] => Sequel.pg_jsonb_wrap(UBID.parse(ubid).to_uuid)).first
```

Comment:
Shouldn't we filter by `id` instead of `name`?
```suggestion
  DeletedRecord.where(Sequel.pg_json(:model_values)["id"] => Sequel.pg_jsonb_wrap(UBID.parse(ubid).to_uuid)).first
```